addons/website_blog/views/snippets/s_dynamic_snippet_blog_posts_preview_data.xml

```diff
@@ -2,61 +2,50 @@
 <odoo>
     <template id="s_dynamic_snippet_blog_posts_preview_data" name="Blogs Preview Data">
         <div class="s_dialog_preview row my-4">
-            <div class="d-flex flex-grow-0 flex-shrink-0 col-3">
-                <figure class="my-3 w-100 s_blog_posts_post">
-                    <a class="s_blog_posts_post_cover position-relative d-flex flex-column shadow-sm overflow-hidden rounded text-decoration-none" href="#">
-                        <div data-name="Cover" class="o_record_cover_container d-flex flex-column o_colored_level o_cc h-100 w-100 bg-600 position-absolute">
+            <div class="col-12 col-sm-6 col-lg-4">
+                <div class="s_blog_posts_post position-relative w-100">
+                    <a class="s_blog_posts_post_cover position-relative d-block h-100 rounded shadow-sm overflow-hidden text-decoration-none text-reset ratio ratio-1x1" href="#">
+                        <div class="o_record_cover_container d-flex flex-column h-100 o_colored_level o_cc w-100 bg-600 position-absolute o_dirty">
```

Comment:
I cleaned this file as much as I could 👍

Reply:
Edit: I restored the file before my cleaning but with your feedback integrated (also removed a useless `text-white` class). This file could indeed be decluttered but as it can be a tricky task (break css and stuff) I think it's safer to do it later 👍